packages/ExternalLink/src/ExternalLink.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import PropTypes from "prop-types";
+import tokens from "@paprika/tokens";
+import NewTabIcon from "@paprika/icon/lib/NewTab";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import * as styled from "./ExternalLink.styles";
+
+const propTypes = {
+  /** Text for aria-label. */
+  ariaText: PropTypes.string,
+
+  /** Link text for showing. */
+  children: PropTypes.string,
+
+  /** Should have an underline, by default it's true */
+  hasUnderline: PropTypes.bool,
+
+  /** Link url for the target. */
+  href: PropTypes.string.isRequired,
+};
+
+const defaultProps = {
+  ariaText: null,
+  children: null,
+  hasUnderline: true,
+};
+
+function ExternalLink(props) {
+  const iconFontSize = 11;
+  const I18n = useI18n();
+  const { ariaText, children, hasUnderline, ...moreProps } = props;
+
+  const getContentText = () => {
+    return children || I18n.t("externalLink.view");
```

Comment:
Will this default value be used in reality?

Reply:
I guess it could be, but shouldn't be encouraged in general. I'm just it says "view" and not "click here" :)

Reply:
I'd make `children` isRequired. 
Having them `null` make not much sense. If the content of the lint is empty, the link is pretty much useless anyway.

Reply:
Ok, i'll remove the fallback text, If they really want it to say "view" they can just pass that as the child....

Reply:
👍 this also means this component doesn't have to depend on l10n